File: Infrastructure/Data/IAuthorRepo.cs
using EntityModels;
using System.Collections.Generic;

namespace Infrastructure.Data
{
    public interface IAuthorRepo : ICRUD<AuthorEM, long>
    {
        IEnumerable<AuthorEM> Get(string term);
        DataTableResponseEM<AuthorEM> GetList(DataTableRequestEM model);
    }
}

File: InfrastructureVB/Data/IAuthorRepo.vb
Imports EntityModels

Namespace Data
    Public Interface IAuthorRepo
        Inherits ICRUD(Of AuthorEM, Long)

        Overloads Function [Get](term As String) As IEnumerable(Of AuthorEM)
        Function GetList(model As DataTableRequestEM) As DataTableResponseEM(Of AuthorEM)
    End Interface
End Namespace